inventory.py

```diff
@@ -25,13 +25,20 @@
 SUBSCRIPTION_NAME = ENV['API_SUBSCRIPTION_NAME']
 API_SCOPE_PREFIX = ENV['API_SCOPE_PREFIX']
 MAX_REQ_ATTEMPTS = ENV['MAX_REQ_ATTEMPTS']
+PER_PAGE_COUNT = ENV['PER_PAGE_COUNT']
 
 UDW_CONN = psycopg2.connect(**ENV['UDW'])
 WAREHOUSE_INCREMENT = ENV['WAREHOUSE_INCREMENT']
 
 
 # Function(s)
 
+def make_audit_logs_api_request(url, params=None):
+    logger.info(f"make_audit_logs_api_request url: {url}")
+    response = API_UTIL.api_call(url, SUBSCRIPTION_NAME, payload=params)
+    return response
```

Comment:
It would be nice if we could combine some of your code here, and maybe the `handle_request_if_failed` functions, with the existing `make_request_using_api_utils`. Not sure how feasible that is, but it seems like there is a lot of duplication now. If we need to change my other paging approach to be more in line with yours, that would be okay.

Reply:
Your code is returning date object after fetching data when you have to page through using link header then you need link headers for that. Here is the code [sample](https://github.com/tl-its-umich-edu/api-utils-python/blob/master/tests/test.py#L80) for that.

Yeah, redundant code but approach for pulling course un/ publish data is different from getting published date. It can be consolidated for the future. 

@zqian what is the long term scope of this project? will this ever being used beyond current situation.

Reply:
Also directly call the  API as like `canvasApiurl?page=34` etc is kind of discouraged by canvas and may not be supported in future, But it's not the problem to solve today

Reply:
Can you point to where this is discoursaged? I did not know `api_utils_python` had paging. We can try to change my code to using that, especially if it will help reduce some duplication here.

Reply:
@ssciolla I have sent you an email regarding that discussion. Based on that it only impacts enrollment API but Canvas [documentation](https://canvas.instructure.com/doc/api/file.pagination.html) suggests only to pull page links from link header